Samples/Windowing/cs-wpf/wpf_packaged_app/DemoPage.xaml.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.Windows.Interop;
+using System.Runtime.InteropServices;
+using Microsoft.UI;
+using Microsoft.UI.Windowing;
+
+namespace wpf_packaged_app
+{
+    /// <summary>
+    /// Interaction logic for DemoPage.xaml
```

Comment:
Remove boilerplate comments like these.
Ok, this will be commented multiple times. But I thought it's better to call them all out so you can easily identify and clean them up. 🙂